app/views/exports/cardinal_tagging_csv_for_custom_pooling.csv.erb

```diff
@@ -0,0 +1,14 @@
+<%
+  csv_array = [['Source Well', 'Volume to add to pool', 'Dest. pool', 'Number of samples', 'Tag index', 'Tag 2 index']]
+  @plate.wells_in_columns.each do |well|
+    csv_array << [
+      "#{@plate.human_barcode}:#{well.location}",
+      nil,
+      nil,
+      rand(12) + 1, # TODO: DPL-067 Determine actual number of samples per well
+      well.aliquots[0].tag_index,
+      well.aliquots[0].tag2_index
+    ] unless well.empty? || well.aliquots.size > 1 # TODO: DPL-067 Verify the assumption that a well only has one aliquot
```

Comment:
Even if the well has multiple aliquots (i.e. not an 'aggregate sample'), they should all have the same tags. Not actually quite sure if the tagging step deals with wells with multiple samples/aliquots, but that's how it should be I think!

Reply:
Ah OK so we should not bother checking for number of aliquots then?  Just take the first one?  Or we should validate that we only have a single unique index across all aliquots?  The latter sounds safer.

Reply:
Yes the latter sounds nice and robust